CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
```

Comment:
We should set up VB and automated npm deployment on this module. I've added these both as issues as they don't need to be done for this change.